ml-agents/mlagents/trainers/settings.py

```diff
@@ -769,6 +769,34 @@ def maybe_init_path(self) -> Optional[str]:
     def run_logs_dir(self) -> str:
         return os.path.join(self.write_path, "run_logs")
 
+    def fix_resume_init(self) -> None:
+        """priorotize explicit command line resume/init over conflicting yaml options.
+        if both resume/init are set at one place use resume"""
+        _non_default_args = DetectDefault.non_default_args
+        if "resume" in _non_default_args:
+            if (
+                "initialize_from" in _non_default_args
```

Comment:
Is this condition needed? initialize_from will not be None if it is specified on the CLI.

Reply:
no it's not!